code/src/index.css

```diff
@@ -5,9 +5,156 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: #000000;
+  color: #ffffff;
 }
 
 code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",monospace;
 }
+
+h1{
+  margin-left: 20px;
+  font-size: 20px;
+}
+
+.album-container{
+  position: relative;
+  place-self: center;
```

Comment:
The albums don’t align at the top when the album titel or artist names takes up two rows. We tried to figure out why in the dev tools and found that when we remove "place-self: center" it aligns correctly according to the design sketch.